src/main/kubeconfig-manager.ts

```diff
@@ -24,14 +24,20 @@ export class KubeconfigManager {
   protected async init() {
     try {
       await this.contextHandler.ensurePort();
-      await this.createProxyKubeconfig();
+      this.tempFile = this.createProxyKubeconfig();
```

Comment:
this might not needed anymore, since it's created on-demand anyway, but maybe doesn't hurt to have it here.